docs-src/content/using-diatheke-sdk/session/audio-output.md

```diff
@@ -189,4 +211,24 @@ let stream = session.streamAudioReplies { (reply) in
 }
 {{< /tab >}}
 
+{{< tab "Java/Android" "java" >}}
+@Override
+public void onNext(DiathekeOuterClass.AudioReply value) {
```

Comment:
Some comments in the code block (similar to examples for other languages) would be nice.